app/models/ftnae/FtneaAuditAnswer.scala

```diff
@@ -0,0 +1,25 @@
+/*
+ * Copyright 2023 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package models.ftnae
+
+import play.api.libs.json.Json
+
+final case class FtneaAuditAnswer(userPresentedText: String, answer: String)
```

Comment:
I don't think it makes sense to the selected language as userPresentedText?

```suggestion
final case class FtnaeAuditAnswer(language: String, List[FtneaAnswers])

final case class FtnaeAnswers(userPresentedText: String, answer: String)
```

Reply:
userPresentedText is the cy or en question that is asked on the form. It is not the language. As per Rakek's suggestion the actual 'language' is now another instance of FtnaeAuditAnswer added to the list.

Reply:
Cool, but as it currently stand the returned content will be in welsh if the user selects welsh. This has to be always be sent to the backend in English only